.travis.yml

```diff
@@ -13,6 +13,7 @@ script:
 - make dist
 - make prepare-test
 - make test-e2e
+- docker build -t dummy-image-for-testing-dockerfile .
```

Comment:
One question - is docker-ce still present as a package in .travis.yml?

Reply:
I don't believe it is, this would fail CI - https://github.com/openfaas/faasd/blob/master/.travis.yml#L10

Reply:
passing https://travis-ci.com/openfaas/faasd/builds/148533376

Reply:
What needs to be done?

Reply:
I thought adding `docker-ce` to packages? https://github.com/openfaas/faas/blob/master/.travis.yml#L13

Reply:
Why? CI seems passing